src/components/detail/DetailTitle.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import './DetailTitle.scss';
+
+const DetailTitle = ({
+  tags,
+  korName,
+  engName,
+  description,
+  colors,
+  volume,
+  price,
+}) => {
+  //setImgSrc([images[0]]);
+  const [dropDown, setDropDown] = useState(false);
+  const [colorIdx, setColorIdx] = useState(0);
+  const handleDropDown = e => {
+    dropDown ? setDropDown(false) : setDropDown(true);
+  };
+
+  const addToCart = e => {
+    handleDropDown();
+    setColorIdx(e.target.id);
+  };
+
+  return (
+    <div className="titleWrapper">
+      <p className="tagWrapper">
+        {tags.map((tag, index) => {
+          return (
+            <span key={index} className="tag">
+              {tag}
+            </span>
+          );
+        })}
+      </p>
+      <p className="title"> {korName} </p>
+      <p className="englishTitle"> {engName}</p>
+      <img
+        src="../images/detail/gray-quotation-marks.webp"
```

Comment:
이미지 경로는 public 하위의 경로를 절대경로로 접근할 수 있습니다!
```suggestion
        src="/images/detail/gray-quotation-marks.webp"
```

Reply:
고쳤습니다!